vendor/brave-ios/Shared/RewardsLogStream.mm

```diff
@@ -7,35 +7,54 @@
 RewardsLogStream::RewardsLogStream(const char* file,
                                    const int line,
                                    const ledger::LogLevel log_level) {
+    
+  this->log_stream = std::make_unique<UnbufferedLogger>([this, file, line, log_level](std::string data){
```

Comment:
C++ question: Is `file` implicitly copied here in this lamdba?  If its not maybe we should copy it into an `std::string` and copy that into the lambda instead